roles/network_plugin/cilium/templates/hubble/deploy.yml.j2

```diff
@@ -56,11 +56,11 @@ spec:
           - mountPath: /etc/hubble-relay
             name: config
             readOnly: true
-          {% if cilium_hubble_tls_generate -%}
+          {%- if cilium_hubble_tls_generate +%}
```

Comment:
HI @y-sera 

The `+%}` is not required in the code :-) .
The `-%}` is enough to make it more readable.

Reply:
OK.  Thanks.
I will add commit Where `+%}` is not needed like line 59, 99 and 145, from `{%- +%}` to `{% -%}.